base/configs/openmrs/frontend_assembly/spa-assemble-config.json

```diff
@@ -1,9 +1,10 @@
 {
+    "coreVersion": "5.6.1-pre.1857",
     "frontendModules": {
         "@openmrs/esm-form-entry-app": "7.1.0"
```

Comment:
@corneliouzbett , please pull in the latest changes, the version for the `@openmrs/esm-form-entry` needs to be `8.0.0`.
Thanks!